xl_auth/templates/public/home.html

```diff
@@ -1,39 +1,64 @@
 <!-- public/home.html -->
 {% extends "layout.html" %}
 {% block content %}
-    <!-- Main jumbotron for a primary marketing message or call to action -->
-    <div class="jumbotron">
-
-        <h1>{{ _('Welcome to xl_auth') }}</h1>
-
-        <p>This is a starter Flask template. It includes Bootstrap 3, jQuery 2, Flask-SQLAlchemy,
-            WTForms, and various testing utilities out of the box.</p>
-        <p><a href="https://github.com/sloria/cookiecutter-flask"
-              class="btn btn-default btn-large">Learn more &raquo;</a></p>
-    </div><!-- /.jumbotron -->
-
     <div class="body-content">
         <div class="row">
+          {% if current_user and current_user.is_authenticated %}
+          <div class="jumbotron">
+
+              <h1>{{ _('Welcome to xl_auth') }}</h1>
+
+              <p>This is a starter Flask template. It includes Bootstrap 3, jQuery 2, Flask-SQLAlchemy,
+                  WTForms, and various testing utilities out of the box.</p>
+              <p><a href="https://github.com/sloria/cookiecutter-flask"
+                    class="btn btn-default btn-large">Learn more &raquo;</a></p>
+          </div><!-- /.jumbotron -->
+          {% elif login_form %}
+            <div class="col-md-12 login-panel panel panel-default">
+              <div class="col-md-8">
+                <div class="header">
+                  <div class="logo-small logotype logotype-black">
+                  </div>
+                  <h2>
+                    xl_auth
```

Comment:
I think these mentions of xl_auth in templates could be `{{ _('xl_auth') }}` instead. And sv-translating it to "Libris Login (alpha)". Will hopefully be deployed to https://login.libris.kb.se this week, so the "xl_auth" labels seems mismatched!